### PR TITLE
Scale head slot correctly for small armor stand

### PR DESCRIPTION
**Pull request by Kas-tle**
Allows for proper scaling of head slot for small armor stand.

See related PR on Geyser:

https://github.com/GeyserMC/Geyser/pull/3210

[GeyserOptionalPack.zip](https://github.com/GeyserMC/GeyserOptionalPack/files/9276159/GeyserOptionalPack.zip)

### CODE REVIEW

**Comment by davchoo** [comment]
This pack seems to work for helmets, but causes `player_head`, `skeleton_skull`, and `carved_pumpkin` to appear way too large. In addition, it seems to have no effect on banners in the head slot.

(Left is Java, Right is Bedrock 1.19.11)
screenshot

**Comment by Kas-tle** [comment]
@davchoo this is not the cause; that occurs without any pack. We discussed somewhat in discord but basically though we'd like to fix this with optionalpack, we cannot.

File: developer_documentation.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ Bedrock armor is defined via attachables, which contain geometry that utilize th
...
...on Java Edition. Consequently, the query `q.is_baby` is applied when an armor stand is small, which is then used to trigger an animation which upscales the head slot by 1.3984. This value was derrived from the [attached chart](https://cdn.discordapp.com/attachments/338267383904993291/417695114249371648/unknown.png) linked in the Blockbench Discord. Since large armor stand head slots have a scale of 0.625 to one block, downscaling by half gives a scale of 0.3125 to one block. The proper scale, per the chart, is 0.437. Thus, the slot must be upscaled by 1.3984 to compensate, which is derrived by dividing 0.437 by 0.3125. Behavior pack scale is not spplied to heads, pumpkins, and banners, which do render in the head slot in vanilla Bedrock. Because the downscaling is not applied to these, a Molang expression is used to set the scale to use. When a head or carved pumpkin is placed in the head slot, the scaling is done at half of 1.3984 to account for the fact that the initial downscaling was never applied. Banner scaling for small armor stands cannot be fixed at this time due to their attachment occuring above the root bone of the model, as they must use the chestplate slot on Bedrock.
```
Comment (davchoo) [inline developer_documentation.md:60]: Some typos.
derrived -> derived
spplied -> applied
occuring -> occuring

**Review by davchoo — CHANGES_REQUESTED** [review]
Besides the typos in the documentation, this PR works fine. Helmets, skulls, and carved_pumpkins seems to be scaled correctly with this pack.

**Review by Camotoy — APPROVED** [review]

**Pull request merged**